image-layout.md

```diff
@@ -53,9 +53,8 @@ afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256/
 ## Blobs
 
 * Object names in the `blobs` subdirectories are composed of a directory for each hash algorithm, the children of which will contain the actual content.
-* A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptor.md#digests-and-verification)), MUST have its content stored in a file under `blobs/<alg>/<hex>`.
-* The character set of the entry name for `<hex>` and `<alg>` MUST match the respective grammar elements described in [descriptor](descriptor.md#digests-and-verification).
-* For example `sha256:5b` will map to the layout `blobs/sha256/5b`.
+* The content of `blobs/<alg>/<encoded>` MUST match the digest `<alg>:<encoded>` (referenced per [descriptor](descriptor.md#digests-and-verification)). For example, the content of `blobs/sha256/5b` MUST match the digest `sha256:5b`.
```

Comment:
I'd rather not use `<encoded>` here, since we [currently require hex there][1].  If/when #654 lands, it can update the `<hex>` reference.

It would also be nice to use a valid descriptor like `sha256:da39a3ee5e6b4b0d3255bfef95601890afd80709`, although since the current master uses `sha256:5b`, that would be a somewhat orthogonal fixup (but it seems like a fairly straightforward fix, so I thought I'd mention it in case you want to pick it up here).

[1]: https://github.com/opencontainers/image-spec/blame/be8ecf486746be4e4076d492df1f8e7c8497e635/descriptor.md#L70